rust/private/rustc.bzl

```diff
@@ -235,7 +235,12 @@ def rustc_compile_action(
         out_dir_env = "OUT_DIR=$(pwd)/{} ".format(out_dir.path)
     else:
         out_dir_env = ""
-    command = '{}{} "$@" --remap-path-prefix="$(pwd)"=__bazel_redacted_pwd'.format(out_dir_env, toolchain.rustc.path)
+        
+    # Similar awkward construction to prepend execroot to the crate-root to set CARGO_MANIFEST_DIR
```

Comment:
Nit: Can you cite either this PR or [my comment](https://github.com/google/cargo-raze/issues/71#issuecomment-433225853) in this code comment? I think this is abstruse enough (specifically the choice of CARGO_MANIFEST_DIR) to confuse later editors/reviewers, and having a back link to the original issue would be helpful.